src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead tr');
+
+thead.addEventListener('click', event => {
+  const index = [...thead.children].indexOf(event.target);
```

Comment:
would be better if you specify what kind of index it is

Reply:
fixed